core/storage/job/gae_models.py

```diff
@@ -36,11 +36,40 @@
 
 
 class JobModel(base_models.BaseModel):
-    """Class representing a datastore entity for a long-running job."""
+    """Class representing a datastore entity for a long-running job.
+
+    Attributes:
+        job_type: The job type.
+        time_queued_msec: The time at which the job was queued, in milliseconds since the epoch.
```

Comment:
@ivange94 -- did the linter not run when you pushed the commit? Some lines are too long. See https://travis-ci.org/oppia/oppia/jobs/176591251

Can you make sure that you run the presubmit checks prior to pushing? If they don't run automatically, you can run the linter manually using `python scripts/pre_commit_linter.py` -- see the top of that file for the relevant arguments to pass in.